### DISCUSSION TITLE
Support Qt Custom Classes

### DISCUSSION THREAD

**Discussion by nsourligas** [fnc12/sqlite_orm#1265] [General]
Hi everyone, 

I want to ask the creator of the library (awesome work by the way) if the Qt framework is fully or partially supported or if anyone in the community has ever tried to use the sqlite_orm library with Qt custom classes like Qt Sqlite, QObject, QDateTime, QVector, QMap, QString, etc. Or if there is a different way to utilize sqlite_orm to support all these Qt custom classes by using a specific feature of the library?

Thanks!

**Comment by fnc12** [comment]
Hi. Thank you for your feedback - really appreciate it. 
1) What is `Qt Sqlite`?
2) How do you expect `QObject`, `QVector` and `QMap` to be supported?
3) `QDateTime` and `QString` can be bound using current API just like any other custom type. Check `examples` folder, especially `enum_binding` and `nullable_enum_binding` examples.
